perllib/FixMyStreet/Cobrand/CentralBedfordshire.pm

```diff
@@ -123,6 +131,35 @@ sub lookup_site_code_config {
 sub open311_extra_data_include {
     my ($self, $row, $h, $contact) = @_;
 
+    if ($contact->email =~ /Jadu/) {
```

Comment:
Do we want to move `$open311_only` creation up above this, then can add the new rows in this if

Reply:
Doh, sorry, my suggestion here broke the code (and test failure showing this) because it needed the change to detail to happen before setting the open311_only. So guess code should be 1. UnitID block, 2. open311_only set, 3. Jadu check.